docs/internal.md

```diff
@@ -54,6 +54,7 @@ VZ:
 
 SSH:
 - `ssh.sock`: SSH control master socket
+- `ssh.config`: SSH config file, printed by `limactl show-ssh --format=config-file` . Not consumed by Lima itself.
```

Comment:
I was confused by this on first reading: it sounds like the file is written only when you run `limactl show-ssh --format=config-file`. But it is actually written ("printed") by the host agent during startup, and this command just prints the name of the config file.